resources/migrations/20160509-create-5.0-csv-tables.up.sql

```
@@ -0,0 +1,285 @@
+                             results_id BIGINT REFERENCES results (id) NOT NULL,
+                             PRIMARY KEY (results_id, id),
+                             date TEXT,
+                             name TEXT,
+                             election_type TEXT,
+                             state_id TEXT,
+                             is_statewide TEXT,
+                             registration_info TEXT,
+                             absentee_ballot_info TEXT,
+                             results_uri TEXT,
+                             polling_hours TEXT,
+                             has_election_day_registration TEXT,
+                             registration_deadline TEXT,
+                             absentee_request_deadline TEXT,
+                             hours_open_id TEXT);
+
+CREATE TABLE v5_0_states (id TEXT NOT NULL,
+                          results_id BIGINT REFERENCES results (id) NOT NULL,
+                          PRIMARY KEY (results_id, id),
+                          name TEXT,
+                          election_administration_id TEXT,
+                          external_identifier_type TEXT,
+                          external_identifier_othertype TEXT,
+                          external_identifier_value TEXT);
+
+CREATE TABLE v5_0_hours_open (id TEXT NOT NULL,
+                              results_id BIGINT REFERENCES results(id) NOT NULL,
+                              schedule_id TEXT NOT NULL,
+                              PRIMARY KEY (results_id, id, schedule_id));
+
+CREATE TABLE v5_0_schedules (id TEXT NOT NULL,
+                             results_id BIGINT REFERENCES results (id) NOT NULL,
+                             PRIMARY KEY (results_id, id),
+                             start_time TEXT,
+                             end_time TEXT,
+                             start_time2 TEXT,
+                             end_time2 TEXT,
+                             is_only_by_appointment TEXT,
```

Ahhh...yes. I did read that but thought that it wasn't necessary for boolean data. But if you want to report an error an include the original value then...`TEXT` it is.